configs/benchmark.yaml

```diff
@@ -0,0 +1,25 @@
+# #############################################################################
```

Comment:
no header for config files? as this are nice to copy, and it would be "annoying" to have to edit/remove the header each time